### ISSUE TITLE
Synctex with multiple .tex files

### ISSUE THREAD

**Issue by Econ808** [James-Yu/LaTeX-Workshop#1737]
**Describe the bug [Required]**

When using synctex from the internal viewer, synctex is unresponsive on most text, except for section headlines and list of contents. 

**To Reproduce**
1. create a Latex project where chapters are included as input.
2. Use synctex on headlines - leads to the main document where the section headlines are defined (see screenshot). This is as expected.
3. Clicking on text anywhere else does nothing. 

**Expected behavior**
When text is clicked I expect synctex to point to the correct .tex file.

**Logs [Required]**
Nothing is logged from this event (to my knowledge).

**Screenshots**

![vscodesynctex](https://user-images.githubusercontent.com/36441764/66329337-a5be7280-e92e-11e9-8455-65d09ed779d7.png)

**Desktop [Required]:**
 - OS:  Windows 10
 - VS Code version: 1.38.1
 - Extension version: 8.2.0

**Are you using LaTeX Workshop with VS Code Remote?**

No

**Comment by jlelong** [comment]
It is designed to work as you expect. So please post a MWE along with the extension logs if you need help.

**Comment by Econ808** [comment]
While making the MWE, I realized that the issue lies with the documents being located in my business OneDrive folder. The path hereto, contains spaces and "special" characters such as "æøå" - I suspect this may cause the issue. Is there a known fix to this?

If the above is should not be an issue, it is probably a *rights issue*...

**Comment by tamuratak** [comment]
Please show us log messages. 

![2018-11-07 20 34 24](https://user-images.githubusercontent.com/10665499/48129417-c4565c80-e2cc-11e8-89de-05b36ef1fa76.png)

1 Please change the theme of VSCode to the default one, and check that you still see this issue.  
2 Please change all the user settings of VSCode to the default ones, and check that you still see this issue.  
3 Please disable all the other extensions except for LaTeX Workshop, and check that you still see this issue. 

Then please do the followings.

1. Open the command palette.
2. Enter `Developer: Toggle Developer Tools` and press enter.
3. Show us messages in the console.

![2018-11-27 20 42 47](https://user-images.githubusercontent.com/10665499/49079823-0a2b9280-f285-11e8-93ee-421fc1870ae8.png)

![2018-11-27 20 43 43](https://user-images.githubusercontent.com/10665499/49079859-29c2bb00-f285-11e8-9446-1be80c11df01.png)

![2018-11-27 20 44 45](https://user-images.githubusercontent.com/10665499/49080094-df8e0980-f285-11e8-9111-2d02affc45f6.png)

Thank you for your cooperation.

**Comment by jlelong** [comment]
I bet the issue lies in the `XXX` so if you need help please the unaltered logs. Your problem mostly comes from a path encoding issue because of the special characters.

**Comment by Econ808** [comment]
Yeah, the path contains the Danish letter "ø" `C:\Users\XXX\OneDrive - Kø... XXX` As it is a business OneDrive I am not comfortable disclosing the entire path, but I guess the above snippet should suffice?
 
Are there any known workarounds?

**Comment by tamuratak** [comment]
We have already been converting the encoding of the path when executing synctex. There is nothing more we can do on our side.

You can change your OneDrive folder. See [support.office.com](https://support.office.com/en-us/article/change-the-location-of-your-onedrive-folder-f386fb81-1461-40a7-be2c-712676b2c4ae?wt.mc_id=SMCPkeyIA&omkt=en-US&ui=en-US&rs=en-US&ad=US).
